hibernate-reactive-core/src/test/java/org/hibernate/reactive/annotations/tests/DisableForClassTest.java

```diff
@@ -0,0 +1,26 @@
+/* Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright: Red Hat Inc. and Hibernate Authors
+ */
+package org.hibernate.reactive.annotations.tests;
+
+
+import org.hibernate.reactive.BaseReactiveTest;
+import org.hibernate.reactive.annotations.DisableFor;
+import org.hibernate.reactive.containers.DatabaseConfiguration;
+
+import org.junit.jupiter.api.Test;
+
+import static org.hibernate.reactive.containers.DatabaseConfiguration.DBType.POSTGRESQL;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+@DisableFor(value = POSTGRESQL, reason = "Reason #1")
+public class DisableForClassTest extends BaseReactiveTest {
```

Comment:
I don't think any of these tests need to extend from `BaseReactiveTest` (I might be wrong)

Reply:
correct. Removed the extension